src/Controls/src/Core/TrimmerConstants.cs

```diff
@@ -6,4 +6,6 @@ class TrimmerConstants
 	internal const string SerializerTrimmerWarning = "Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.";
 
 	internal const string NativeBindingService = "This method properly handles missing properties, and there is not a way to preserve them from this method.";
+
+	internal const string XamlLoadingTrimmerWarning = "Loading XAML at runtime might require types that cannot be statically analyzed. Make sure all of the required types are preserved.";
```

Comment:
Will I get this warning in an app which doesn't call any of the affected public APIs, but if I just enable the feature switch?
If so, then this should mention the feature switch and how to disable it (as that's the way to "fix" this).

Or maybe a better way to put it - I create an app, and enable this feature. Then I AOT the app. Is there any case where the behavior of the app would change after AOT in this case? Especially is there a way to hit one of the new exceptions? If the answer is yes, then we should make it to produce a warning. We do this for example for startup hooks - if you enable startup hooks, you will get trim warning. It may be that at runtime you don't run startup hooks and so you never hit the problem - but we "play it safe" and produce a warning regardless.
I would expect similar behavior here...

Reply:
It's also a good idea to add the `Url` property to the RUC so that we have a way to direct users to a longer and much more detailed description of the problem and how to solve it. In this case one such solution would be to discuss XamlC for example.

Reply:
> Will I get this warning in an app which doesn't call any of the affected public APIs, but if I just enable the feature switch?

No, this warning will only show up in apps that call `.LoadFromXaml(...)` explicitly.

> It's also a good idea to add the Url property to the RUC so that we have a way to direct users to a longer and much more detailed description of the problem and how to solve it.

I think we'll need to revisit this later. We currently don't have any documentation or guidelines that we could point to.